<launch>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lidar_200)/rviz_cfg/mapping.rviz"/>
    <node name="mapping" pkg="lidar_200" type="mapping" output="screen"> 
        <!-- Optional-->
        <param name="PARA_GRID_SIZE_MAP" type="double" value="1.0" />
        <param name="PARA_MIN_NOVE" type="double" value="2.0" />
        <param name="PARA_MAX_PTS_PER_MAP_GRID" type="int" value="100" />
        <param name="PARA_MIN_PTS_PER_GROUND_GRID" type="int" value="2" />
        <param name="PARA_GRID_SIZE_GROUND" type="double" value="2.0" />
        <param name="PARA_VOXEL_LEAF" type="double" value="1.0"/>
        <!-- Necessary-->
        <param name="PARA_MIN_Z_MAP" type="double" value="-0.6" />
        <param name="PARA_MAX_Z_MAP" type="double" value="2.2" />
        <param name="PARA_SENSOR_HEIGHT" type="double" value="1.75" />
        <param name="PARA_DIR_MAP" type="string" value= "/media/spcomen21/WD5T/Experiment/LiDAR-200/MulRan/DCC01/map/"/>
    </node>
</launch>
